hybrid/decomposers.py

```diff
@@ -828,6 +940,25 @@ def _make_cubic_lattice(dimensions):
                                         ])
     return cubic_lattice_graph
 
+def _make_kings_lattice(dimensions, is_open=(True,True)):
```

Comment:
Since you use `is_open` for integer arithmetic, perhaps use `int` instead of `bool`? Alternatively, I'd convert to int explicitly in the code below.